openassessment/assessment/models/staff.py

```diff
@@ -51,6 +51,35 @@ def is_cancelled(self):
         return bool(self.cancelled_at)
 
     @classmethod
+    def get_workflow_statistics(cls, course_id, item_id):
+        """
+        Returns the number of graded, ungraded, and in-progress submissions there are for staff grading.
```

Comment:
Nit: "submissions that are present for staff grading"? The wording of this seems a bit clunky on first pass.